tool/tctl/common/inventory_command.go

```diff
@@ -50,15 +66,23 @@ func (c *InventoryCommand) Initialize(app *kingpin.Application, config *service.
 	c.inventoryStatus = inventory.Command("status", "Show inventory status summary")
 	c.inventoryStatus.Flag("connected", "Show locally connected instances summary").BoolVar(&c.getConnected)
 
+	c.inventoryList = inventory.Command("list", "List teleport instance inventory").Alias("ls")
+	c.inventoryList.Flag("version", "Filter output by version").StringVar(&c.version)
+	c.inventoryList.Flag("services", "Filter output by service").StringVar(&c.services)
```

Comment:
Would it be worth including the valid services in the help message? It's sometimes frustrating when you're trying to `tctl get` a resource and you cant remember if the resource is `kubeService` or `kube_service` - I could see similar frustrations here not knowing the valid options.